sa/saro.go

```diff
@@ -1402,3 +1402,96 @@ func (ssa *SQLStorageAuthorityRO) GetSerialsByAccount(req *sapb.RegistrationID,
 		return stream.Send(&sapb.Serial{Serial: row.Serial})
 	})
 }
+
+// CheckIdentifiersPaused takes a slice of identifiers of type "dns" and returns
```

Comment:
This doesn't explicitly take a slice of identifiers of type "dns", it's just because we only have the DNS IdentifierType configured in `//identifier/identifier.go` that this statement is true.

Reply:
Whoops, this should have been rewritten when I added support for mixed identifiers.